Docs: updates for new and archived branches (#691)
ACCESS-NRI/access-esm1.6-configs: documentation/docs/pages/index.md

`release-{scenario}[+{modifier}]`

where release signifies this is the release branch that is tested, versioned and ready for use, scenario is the base experimental design with optional modifiers. All configurations are global with nominal 1 degree resolution.

Some examples of possible values of the specifiers:

- scenario: `historical`, `preindustrial`, `4xCO2`

> - modifier: `concentration`, `emissions`

Reviewer: ```suggestion
- modifier: `concentrations`, `emissions`
```

> where scenario is typically a CMIP experiment identifier and concentration and emissions describe the CO2 cycling protocol.

Reviewer: ```suggestion
where scenario is typically a CMIP experiment identifier and `concentrations` and `emissions` describe the CO2 cycling protocol.
```

> These names do not exactly match [CMIP7 experiment names](https://airtable.com/embed/apphXCUgASIeT6jCz/shrCs1cSWzQRV0v4i/tblbT6XAdQYOCMXu7), we use `+concentration` modifier for concentration driven experiments (these are more similar to CMIP6 experiments, and do not have `esm` in their names) and `+emissions` for emissions driven experiments (those with the `esm-` prefix in their CMIP7 name).

Author: We can revisit after #697

> These names do not exactly match [CMIP7 experiment names](https://airtable.com/embed/apphXCUgASIeT6jCz/shrCs1cSWzQRV0v4i/tblbT6XAdQYOCMXu7), we use `+concentration` modifier for concentration driven experiments (these are more similar to CMIP6 experiments, and do not have `esm` in their names) and `+emissions` for emissions driven experiments (those with the `esm-` prefix in their CMIP7 name).

Reviewer: ```suggestion
These names do not exactly match [CMIP7 experiment names](https://airtable.com/embed/apphXCUgASIeT6jCz/shrCs1cSWzQRV0v4i/tblbT6XAdQYOCMXu7), we use `+concentrations` modifier for concentration driven experiments (these are more similar to CMIP6 experiments, and do not have `esm` in their names) and `+emissions` for emissions driven experiments (those with the `esm-` prefix in their CMIP7 name).
```

#### Supported configurations

All available configurations are browsable under [the list of release branches](https://github.com/ACCESS-NRI/access-esm1.6-configs/branches/all?query=release-). There are currently no released configurations.

These configurations are under active development:

- [dev-1pctCO2](https://github.com/ACCESS-NRI/access-esm1.6-configs/tree/dev-1pctCO2)
- [dev-1pctCO2-bgc](https://github.com/ACCESS-NRI/access-esm1.6-configs/tree/dev-1pctCO2-bgc)
- [dev-1pctCO2-rad](https://github.com/ACCESS-NRI/access-esm1.6-configs/tree/dev-1pctCO2-rad)
- [dev-4xCO2+concentrations](https://github.com/ACCESS-NRI/access-esm1.6-configs/tree/dev-4xCO2%2Bconcentrations)
- [dev-amip](https://github.com/ACCESS-NRI/access-esm1.6-configs/tree/dev-amip)

> - [dev-flat10](https://github.com/ACCESS-NRI/access-esm1.6-configs/tree/dev-flat10)

Reviewer 2: Should we rename to `dev-flat10+emissions` considering #697 and do the renaming before pushing this documentation change so we can change in the doc at the same time?

Author: I think fix this after #697

](https://github.com/ACCESS-NRI/access-esm1.6-configs/compare/dev-preindustrial+concentrations..dev-1pctCO2
)

There were some configurations previously under development that have now been archived. 

> These were `+CN` only, without a phosporus cycle in the land and are available for reference:

Reviewer 2: ```suggestion
These were `+CN` only, i.e. including the carbon and nitrogen cycles but excluding the phosphorus cycle in the land and are available for reference:
```
Or do we want a table of all modifiers somewhere with their definition? Might be handy?

- [archive-1pctCO2-rad+CN](https://github.com/ACCESS-NRI/access-esm1.6-configs/releases/tag/archive-1pctCO2-rad%2BCN)
- [archive-1pctCO2+CN](https://github.com/ACCESS-NRI/access-esm1.6-configs/releases/tag/archive-1pctCO2%2BCN)
- [archive-4xCO2+concentrations+CN](https://github.com/ACCESS-NRI/access-esm1.6-configs/releases/tag/archive-4xCO2%2Bconcentrations%2BCN)
- [archive-flat10+CN](https://github.com/ACCESS-NRI/access-esm1.6-configs/releases/tag/archive-flat10%2BCN)
- [archive-historical+concentrations+CN](https://github.com/ACCESS-NRI/access-esm1.6-configs/releases/tag/archive-historical%2Bconcentrations%2BCN)
- [archive-preindustrial+concentrations+CN](https://github.com/ACCESS-NRI/access-esm1.6-configs/releases/tag/archive-preindustrial%2Bconcentrations%2BCN)
- [archive-preindustrial+emissions+CN](https://github.com/ACCESS-NRI/access-esm1.6-configs/releases/tag/archive-preindustrial%2Bemissions%2BCN)

If you would like to run these archived configurations, please use the 
[ACCESS-Hive forum for support](https://forum.access-hive.org.au/new-topic?category=esm&tags=access-nri-help) on how to 

> update these configurations up with the latest changes in the configurations under active development.

Reviewer 2: Do we really want to provide that kind of support for archived configurations?

Reviewer 2: Would we be better providing generic instructions and let the community figure it out? Instead of saying people will get personalised help whenever they want to revive an archive configuration.

Author: I didn't think anyone would take up this offer, so wasn't worried too much.

I could generalise a bit, ...

_If you need assistance in using git to update these configurations, please use the ACCESS-Hive forum_ ?

Reviewer 2: Yes, I think that's better. Maybe "ask on the ACCESS-Hive forum" instead of "use".
